setup.py

```diff
@@ -821,7 +821,13 @@ def use_educated_guess(self):
     author_email="ntamas@gmail.com",
     ext_modules=[igraph_extension],
     package_dir={"igraph": "src/igraph"},
-    packages=["igraph", "igraph.app", "igraph.drawing", "igraph.remote"],
+    packages=[
+        "igraph",
+        "igraph.app",
+        "igraph.drawing",
+        "igraph.remote",
+        "igraph.constructors",
+        ],
```

Comment:
Nowadays I tend to delegate all formatting-related decisions to [`black`](https://black.readthedocs.io/en/stable/) so I could free up some mental capacity to focus on more important things. Eventually, all of `python-igraph`'s source code will be piped through `black` and there will be a pre-commit hook that runs `black` on changed files before committing. I'm just mentioning this so you won't put too much effort into formatting as `black` will take care of it anyway. 

In this particular case, `black` will dedent the closing square bracket so it lines up with `packages=` above.

Reply:
you're right, sorry about that, just habit...

Reply:
No problem, just wanted to mention that you shouldn't spend too much time on code formatting :)